packages/core/test/mocks/mockHandleProvider.ts

```diff
@@ -0,0 +1,11 @@
+import { Cardano } from '../../src';
+
+export const Handle = {
```

Comment:
Once https://github.com/input-output-hk/cardano-js-sdk/pull/720 is merged, please hoist this to `util-dev/src/mockProviders` and also rename the variable
```suggestion
export const handleResolution = {
```